osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/ReverseArrowPiece.cs

```diff
@@ -8,12 +8,15 @@
 using osu.Game.Beatmaps.ControlPoints;
 using osu.Game.Graphics.Containers;
 using osu.Game.Skinning;
+using osu.Framework.Allocation;
+using osu.Game.Rulesets.Objects.Drawables;
 
 namespace osu.Game.Rulesets.Osu.Objects.Drawables.Pieces
 {
     public class ReverseArrowPiece : BeatSyncedContainer
     {
-        public bool ShouldFollowBeats = true;
+        [Resolved]
+        private DrawableHitObject drawableSlider { get; set; }
```

Comment:
It isn't slider that's resolved. it's the repeat...

Reply:
i would hope that's what you want?

Reply:
nah, it's the repeat that's hit. the first commit had the repeat set the bool.

Reply:
yeah. so just the naming was wrong. that's fine.